### PR TITLE
entgql: add goModel directive to generated enums

### PR DESCRIPTION
**Pull request by a8m**
Next PR, will add support for mapping fields with custom GoType to `goModel`.

### CODE REVIEW

**Comment by giautm** [comment]
Hey, to make sure this working. please remove the mapping in the function `genModels()`

File: entgql/schema.go [hunk 1]
```diff
@@ -339,3 +330,16 @@ func printSchema(schema *ast.Schema) string {
...
 }
+
+func goModel(ident string) *ast.Directive {
+	return &ast.Directive{
+		Name:     "goModel",
+		Location: ast.LocationObject,
+		Arguments: ast.ArgumentList{
+			{
+				Name:  "model",
+				Value: &ast.Value{Kind: ast.StringValue, Raw: ident},
```
Comment (giautm) [inline entgql/schema.go:341]:
```suggestion
				Value: &ast.Value{
					Kind: ast.StringValue,
					Raw: ident,
				},
```

File: entgql/schema_models.go [hunk 2]
```diff
@@ -50,30 +52,22 @@ func (e *schemaGenerator) genModels() (map[string]string, error) {
...
-			switch {
-			case field.IsOther() || (field.IsEnum() && field.HasGoType()):
-				goType = fmt.Sprintf("%s.%s", field.Type.RType.PkgPath, field.Type.RType.Name)
-			case field.IsEnum():
-				goType = fmt.Sprintf("%s/%s", e.graph.Package, field.Type.Ident)
-			default:
+			goType, ok := e.fieldGoType(field)
+			if !ok {
 				continue
 			}
```
Comment (giautm) [inline entgql/schema_models.go:63]:
```suggestion
			if !field.IsOther() {
				continue
			}
			goType := fmt.Sprintf("%s.%s", field.Type.RType.PkgPath, field.Type.RType.Name)
```

Comment (giautm) [reply entgql/schema_models.go:63]: and update `TestModifyConfig_todoplugin_relay`

**Review by giautm — APPROVED** [review]
Hey, I think we can skip the update unit-tests part.

**Comment by a8m** [comment]
> Hey, I think we can skip the update unit-tests part.

Not sure what do you mean by that.

**Pull request merged**